### PR TITLE
non-critical-infra/owncast: init

### PR DESCRIPTION
**Pull request by mweinelt**
Deploys owncast under live.nixos.org.

### CODE REVIEW

**Comment by JulienMalka** [comment]
(Can't merge this bc needs a review from infra-build)

**Review by vcunat — APPROVED** [review]

**Comment by vcunat** [comment]
I think that's just because of the DNS change.

**Comment by vcunat** [comment]
I see DNS live already, nothing wrong here at a glance (and non-critical anyway).  So why block it?

**Pull request merged**
